test/shell/test_coverage_scalatest_resources.sh

```diff
@@ -5,7 +5,9 @@ dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 runner=$(get_test_runner "${1:-local}")
 
 test_coverage_succeeds_resource_call() {
-    bazel coverage //test/coverage_scalatest_resources/consumer:tests
+    bazel coverage \
```

Comment:
@fmeum thanks for contribution! Why you had to change this test? Could add new one with `--instrumentation_filter` and leave this one as is. As I understand coverage should work without `--instrumentation_filter` as well.